lava-slave/Dockerfile

```diff
@@ -1,40 +1,6 @@
-FROM bitnami/minideb:stretch
+FROM baylibre/lava-slave-docker-image:latest
```

Comment:
Similar to master, please rename to baylibre/lava-slave-base and use specific version.

Reply:
use: baylibre/lava-slave

Reply:
as discussed in meeting, changed to lava-slave-base:latest